src/main/java/data_access/APIDataAccessObject.java

```diff
@@ -0,0 +1,23 @@
+// Author: André
+
+package data_access;
+
+import api.JsonCollegeScorecardDB;
+import entity.UniversityFactory;
+import entity.University;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import use_case.search.SearchUserDataAccessInterface;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class APIDataAccessObject implements SearchUserDataAccessInterface {
+
+    private final JsonCollegeScorecardDB database = new JsonCollegeScorecardDB();
+
+    @Override
+    public JSONObject searchQuery(String searchparameters) {
+        return database.basicQuery(searchparameters, "fields=id,school.name,school.state,school.city,admissions.admission_rate.overall,cost.tuition.in_state,cost.tuition.out_of_state,admissions.sat_scores.average.overall,admissions.act_scores.midpoint.cumulative,school.school_url");
+    }
```

Comment:
Eventually, you should change this to return an actual data entity (i.e the university data entity) rather than the Json entity to better conform with CA and SOLID. That way, you can have multiple different implementations of this data access object down the road if needed